control-plane-operator/controllers/hostedcontrolplane/etcd/reconcile.go

```diff
@@ -262,7 +262,7 @@ func buildEtcdContainer(p *EtcdParams, namespace string) func(c *corev1.Containe
 			},
 			{
 				Name:  "QUOTA_BACKEND_BYTES",
-				Value: strconv.FormatInt(p.StorageSpec.PersistentVolume.Size.Value(), 10),
```

Comment:
Having a bigger quota than the PVC will not hit costs in the SRE part, but the PVC with a bigger size will create a bigger volume which maybe doesn't care having one or two but in bulk maybe the costs will raise up.

I've decoupled them because in a day 2 operation, it's possible to modify the PVC Size without issues but, the StatefulSet will not allow to modify the quota in the Spec. Make sense?

Reply:
https://github.com/openshift/hypershift/pull/2549#discussion_r1191195805